src/components/tooltip/tooltip.svelte

```diff
@@ -173,7 +177,6 @@
   }
 
   .leo-tooltip .tooltip {
-    width: max-content;
```

Comment:
I have a feeling that removing this is going to reintroduce the layout shift which was fixed by https://github.com/brave/leo/pull/488